test-generator/src/generator_help.cr

```diff
@@ -91,6 +91,22 @@ class GeneratorHelp
     input.tr("-", "_").camelcase
   end
 
+  def function(input : JSON::Any, arg_types : Array(String))
```

Comment:
This method should likely be placed in the generator_plugins file istead since it is an addition to the templates.

Reply:
Sounds good. I'll also move the two functions I had at the top of the template there too.